includes/specials/SpecialSearch.php

```diff
@@ -220,123 +232,82 @@ public function showResults( $term ) {
 		$search->showRedirects = $this->searchRedirects; // BC
 		$search->setFeatureData( 'list-redirects', $this->searchRedirects );
 		$search->prefix = $this->mPrefix;
-		$term = $search->transformSearchTerm($term);
+		$term = $search->transformSearchTerm( $term );
 
-		Hooks::run( 'SpecialSearchSetupEngine', array( $this, $this->profile, $search ) );
+		Hooks::run( 'SpecialSearchSetupEngine', [ $this, $this->profile, $search ] );
 
 		$this->setupPage( $term );
 
 		$out = $this->getOutput();
 
-		// Wikia change /Begin (ADi)
-		if(($search instanceof SearchErrorReporting) && $search->getError()) {
```

Comment:
As far as I can tell SearchErrorReporting does not exist and cannot be returned from `		$search = $this->getSearchEngine()`